Script/menu.js

```diff
@@ -112,4 +112,30 @@ form.addEventListener('submit', (e) => {
   if (isEmailValid) {
     form.submit();
   }
-});
\ No newline at end of file
+});
+// Local storage
+const dataStorage = {
+  full_name: '',
+  email_address: '',
+  message_text: '',
+};
+function setStyles() {
+  const fullName = JSON.parse(localStorage.getItem('form_data')).full_name;
+  const emailAddress = JSON.parse(localStorage.getItem('form_data')).email_address;
+  const messageText = JSON.parse(localStorage.getItem('form_data')).message_text;
+  document.querySelector('#fullname').value = fullName;
+  document.querySelector('#email').value = emailAddress;
+  document.querySelector('#message').value = messageText;
+}
+setStyles();
+function populateStorage() {
+  dataStorage.full_name = document.querySelector('#fullname').value;
+  dataStorage.email_address = document.querySelector('#email').value;
+  dataStorage.message_text = document.querySelector('#message').value;
+  const storedData = JSON.stringify(dataStorage);
+  localStorage.setItem('form_data', storedData);
+}
+
+document.querySelector('#fullname').addEventListener('change', populateStorage);
+document.querySelector('#email').addEventListener('change', populateStorage);
+document.querySelector('#message').addEventListener('change', populateStorage);
```

Comment:
Your local storage functionality is not working properly. After I reload the page, the data is lost. Consider fixing it. Also there is a TypeError in your code. Consider fixing it also.
![image](https://user-images.githubusercontent.com/33268927/174323611-69e615d4-d092-4308-9d0b-df0916531930.png)